### PR TITLE
Improve Quickstart iOS - Calling APIs

### PR DESCRIPTION
**Pull request by cocojoe**

### CODE REVIEW

**Comment by cocojoe** [comment]
Will rebase once https://github.com/auth0/docs/pull/5203 is merged

**Comment by cocojoe** [comment]
Have rebased this now that #5203 is merged

File: articles/quickstart/native/ios-swift/04-calling-apis.md [hunk 1]
```diff
@@ -17,26 +19,30 @@ budicon: 546
...
-${snippet(meta.snippets.setup)}
+To retrieve an access token that is authorized to access your API, you need to specify the **API Identifier** value you created in the [Auth0 APIs Dashboard](https://manage.auth0.com/#/apis).
 
-Then present the hosted login screen, like this:
+Present the Hosted Login Page:
 
 ```swift
 // HomeViewController.swift
 
+let APIIdentifier = API_IDENTIFIER // Replace with the API Identifier value you created
```
Comment (lbalmaceda) [inline articles/quickstart/native/ios-swift/04-calling-apis.md]: I'd use `let APIIdentifier = "API_IDENTIFIER"`

File: articles/quickstart/native/ios-swift/04-calling-apis.md [hunk 2]
```diff
@@ -17,26 +19,30 @@ budicon: 546
...
+
 Auth0
     .webAuth()
-    .audience("https://${account.namespace}/userinfo")
+    .audience(APIIdentifier)
     .scope("openid profile")
```
Comment (lbalmaceda) [inline articles/quickstart/native/ios-swift/04-calling-apis.md]: how about requesting an API scope as well?

**Comment by lbalmaceda** [comment]
@cocojoe Looks good. I'd ask Jeff to review the urls or paths though

File: articles/quickstart/native/ios-swift/04-calling-apis.md [hunk 3]
```diff
@@ -17,34 +19,39 @@ budicon: 546
...
 
-This example shows how to use the `accessToken` value. 
+Present the Hosted Login Page:
 
 ::: note
 Depending on the standards in your API, you configure the authorization header differently. The code below is just an example.
 :::
+let APIIdentifier = API_IDENTIFIER // Replace with the API Identifier value you created
+
+    .audience(APIIdentifier)
```
Comment (lbalmaceda) [inline articles/quickstart/native/ios-swift/04-calling-apis.md]: is this `dot` intended?

Comment (cocojoe) [reply articles/quickstart/native/ios-swift/04-calling-apis.md]: Think this got wrecked during conflict resolution, have restored the snippet.

**Review by lbalmaceda — APPROVED** [review]

**Comment by cocojoe** [comment]
Had forgot about this 😆 now I have to rebase *sigh*

**Pull request merged**